releasenotes/notes/logs-sender-destination-add-telemetry-31233181bb0d24fe.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add new `logs_client_http_destination` metric to track HTTP payloads submitted by host and error type.
```

Comment:
You can omit the reno in this PR since telemetry is not customer facing - and therefore does not need to be in the changelog. You can set the `no-changelog` tag to suppress the Github check.

Reply:
Oh ok, removed